cardinal/server/swagger.yml

```diff
@@ -0,0 +1,300 @@
+            consumes:
+                - application/json
+            produces:
+                - application/json
+            operationId: read
+            responses:
+                '200':
+                    description: list of read endpoints
+                    schema:
+                        $ref: '#/definitions/ReadListEndpoints'
+                '400':
+                    description: Invalid read request
+    /query/receipts/submit:
+        get:
+            summary: Get transaction receipts from Cardinal
+            description: Get transaction receipts from Cardinal
+            consumes:
+                - application/json
+            produces:
+                - application/json
+            operationId: receipts
+            parameters:
+                - name: receiptRequestBody
+                  required: true
+                  in: body
+                  schema:
+                      $ref: '#/definitions/ReceiptsRequest'
+            responses:
+                '200':
+                    description: successful operation
+                    schema:
+                        $ref: '#/definitions/ReceiptsReply'
+                '400':
+                    description: Invalid transaction request
+
+definitions:
+    CreatePersonaTransaction:
+        type: object
+        required:
+            - PersonaTag
+            - SignerAddress
+        properties:
+            PersonaTag:
+                type: string
+            SignerAddress:
+                type: string
+    CreatePersonaTransactionResult:
+        type: object
+        required:
+            - Success
+        properties:
+            Success:
+                type: boolean
+    ReadPersonaSignerRequest:
+        type: object
+        required:
+            - PersonaTag
+            - Tick
+        properties:
+            PersonaTag:
+                type: string
+            Tick:
+                type: integer
+                format: int64
+    ReadPersonaSignerResponse:
+        type: object
+        required:
+            - Status
+            - SignerAddress
+        properties:
+            Status:
+                type: string
+            SignerAddress:
+                type: string
+    ReadListEndpoints:
+        type: object
+        required:
+            - tx_endpoints
+            - query_endpoints
+        properties:
+            tx_endpoints:
+                type: array
+                items:
+                    type: string
+            query_endpoints:
+                type: array
+                items:
+                    type: string
+        items:
+            type: string
+    TxReply:
+        required:
+            - tx_hash
+            - tick
+        type: object
+        properties:
+            tx_hash:
+                type: string
+            tick:
+                type: integer
+                format: int64
+    TxRequest:
+        required:
+            - personaTag
```

Comment:
Not 100% sure, but this should probably be "persona_tag"

Reply:
Actually it's PersonaTag. All the names should be capitalized. I will correct it now. When I get to implementing and testing each route, I will automatically have to make these correct so for now if it's rough i think it's fine.